src/main/java/com/automatedtest/sample/pages/SettingPage.java

```diff
@@ -103,4 +107,38 @@ public String getSelectedRadioSpokenValue() {
         }
         return value;
     }
+
+    public void scrollToBottom() {
+        JavascriptExecutor js = ((JavascriptExecutor) this.driver);
+        js.executeScript("window.scrollTo(0, document.body.scrollHeight)");
+        this.wait.sleep(1);
+    }
+
+    public void scrollToTop() {
+        JavascriptExecutor js = ((JavascriptExecutor) this.driver);
+        js.executeScript("window.scrollTo(0, 0)");
+        this.wait.sleep(1);
+    }
+
+    public void takeSnapShot(int id) throws Exception {
+        TakesScreenshot scrShot = ((TakesScreenshot) this.driver);
+        File SrcFile = scrShot.getScreenshotAs(OutputType.FILE);
+
+        if (id == 1) {
```

Comment:
you don't have to do it like this. Just pass the image output path as parameter. And then we can compare it later using the path